cpu/cortexm_common/periph/pm.c

```diff
@@ -24,7 +24,7 @@
 #include "cpu.h"
 #include "periph/pm.h"
 
-#ifndef FEATURE_PERIPH_PM
+#ifndef MODULE_PM_LAYERED
```

Comment:
Shouldn't this be `#ifndef PROVIDES_PM_SET_LOWEST`? Because any other CPU *could* in theory define `pm_set_lowest`

Reply:
The problem with using `PROVIDES_PM_SET_LOWEST` here is, that it does not clearly differentiate from the fallback implementation in `drivers/periph_common/pm.c` (which uses the that guard).

But true, this line needs to be changed, how about
```c
#if !defined(MODULE_PM_LAYERED) && !defined(PROVIDES_PM_SET_LOWEST_CORTEXM)
```

So we have 4 possible cases:
- CPU branch uses `pm_set_lowest()` from `periph_common` -> do nothing
- CPU uses 'set_lowest()` defined by architecture -> define `PROVIDES_PM_SET_LOWEST` by CPU architecture, do nothing else
- CPU uses `pm_layered` -> simply use that module, it defines `PROVIDES_PM_SET_LOWEST` in any case (if not done already by the CPU architecture config)
- CPU wants to use its own implementation -> CPU defines `PROVIDES_PM_SET_LOWEST_CORTEXM', and do not use `pm_layered`